src/screens/Authentication/PhoneNumber/PhoneNumber.tsx

```diff
@@ -57,6 +64,13 @@ function PhoneNumberScreen({
 
   const onError: SubmitErrorHandler<FormValues> = errors => console.log(errors);
```

Comment:
not sure if we need this? can probably remove as it seems like errors are being caught in the onSubmit function?